eservice/pdo/eservice/scripts/EServiceEnclaveInfoCLI.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import argparse
+
+import pdo.common.config as pconfig
+import pdo.common.logger as plogger
+
+import pdo.eservice.pdo_helper as pdo_enclave_helper
+import pdo.eservice.pdo_enclave as pdo_enclave
+import logging
+logger = logging.getLogger(__name__)
+
+## XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+## XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+def LocalMain(config) :
+    # enclave configuration is in the 'EnclaveConfig' table
+    try :
+        logger.debug('initialize the enclave')
+        pdo_enclave_helper.initialize_enclave(config.get('EnclaveModule'))
+        
+        logger.info("MRENCLAVE: " + pdo_enclave.get_enclave_measurement())
+        logger.info("BASENAME: " + pdo_enclave.get_enclave_basename())
+        logger.info("EPID Group: " + pdo_enclave.get_enclave_epid_group())
+        pdo_enclave.dump_enclave_ias_settings()
+    except Error as e :
```

Comment:
Why use the string append for logger messages? We have printf-style formatting for the messages that is used consistently through the code.